### PR TITLE
Fix for #998 - Quoting service not using most recent endpoint

### PR DESCRIPTION
**Pull request by oderayi**
Related issue: https://github.com/mojaloop/project/issues/998

Ensures that both `getQuotePartyEndpoint` and `getParticipantEndpoint` in `database.js` returns only the active endpoint.

### CODE REVIEW

**Comment by lewisdaly** [comment]
I'm not seeing any tests here - perhaps this would be a good opportunity to start adding a few integration tests to this codebase.

I'm also not too familiar with this part of the code, so I think I'll leave PR review to my more learned colleagues.

**Comment by elnyry** [comment]
> I'm not seeing any tests here - perhaps this would be a good opportunity to start adding a few integration tests to this codebase.
> 
> I'm also not too familiar with this part of the code, so I think I'll leave PR review to my more learned colleagues.

@lewisdaly I discussed this with Steven.. We decided given the context of this service, its better to follow up the tests via a separate story

File: docker-compose.yml [hunk 1]
```diff
@@ -7,13 +7,5 @@ services:
           - mojaloop/quoting-service
           - quoting-service
       container_name: qs_quoting-service
-      links:
-        - mysql
-        - kafka
```
Comment (rmothilal) [inline docker-compose.yml]: Hey Steven why did you remove this?

Comment (oderayi) [reply docker-compose.yml]: @rmothilal I removed them because `kafka` and `mysql` services are not defined within that `docker-compose.yml` file.   So linking to them doesn't seem necessary.
